### PR TITLE
add GTK4 tutorial links to documentation page

### PR DESCRIPTION
**Pull request by xflywind**
close https://github.com/nim-lang/website/pull/250

solve the confilcts

### CODE REVIEW

File: jekyll/documentation.html [hunk 1]
```diff
@@ -147,6 +147,12 @@ <h3>
         who have never programmed before, or are new to the language.
         </p>
       </div>
+
```
Comment (narimiran) [inline jekyll/documentation.html:150]: This needs to be in a separate `row`, i.e. you need to put it in `<div class="row">` outside of the current 2nd row.

File: jekyll/documentation.html [hunk 2]
```diff
@@ -147,6 +147,12 @@ <h3>
         who have never programmed before, or are new to the language.
         </p>
       </div>
+      
+      <div class="pure-u pure-u-md-1-3">
+      <h3><a href="http://ssalewski.de/gtkprogramming.html">GTK4 for Graphical User Interfaces</a></h3>
+      <p>An in-depth guide by Dr. Stefan Salewski.</p>
```
Comment (narimiran) [inline jekyll/documentation.html:153]: This is not a good description.

File: jekyll/documentation.html (hunk 1 above)
Comment (xflywind) [reply jekyll/documentation.html:150]: I see, thanks

File: jekyll/documentation.html (hunk 2 above)
Comment (xflywind) [reply jekyll/documentation.html:153]: ok

Comment (xflywind) [reply jekyll/documentation.html]: fixed

File: jekyll/documentation.html (hunk 1 above)
Comment (xflywind) [reply jekyll/documentation.html]: fixed

**Pull request closed without merging**